test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/preference/page/OpenShift3PreferencePage.java

```diff
@@ -45,5 +45,9 @@ public void setOCLocation(String path) {
 	public void clearOCLocation() {
 		setOCLocation("");
 	}
+	
+	public LabeledText getOCLocation() {
+		return new LabeledText(OpenShiftLabel.TextLabels.OC_LOCATION);
```

Comment:
@odockal you should distinguish between linux and win: on Linux is label "'oc' executable location", but on windows is "'oc.exe' executable location"

Reply:
@jkopriva got it, thanks!